app/root.tsx

```diff
@@ -341,7 +341,7 @@ const Mode = () => {
 			<div className="flex items-center gap-2 relative">
 				<Sun className="h-[1.2rem] w-[1.2rem] rotate-0 scale-100 transition-all dark:-rotate-90 dark:scale-0" />
 				<Moon className="absolute h-[1.2rem] w-[1.2rem] rotate-90 scale-0 transition-all dark:rotate-0 dark:scale-100" />
-				{theme === Theme.DARK ? 'Dark' : 'Light'} Mode
+				{theme === Theme.LIGHT ? 'Dark' : 'Light'} Mode
```

Comment:
i never know if these types of UI should show the current state or the "desired" state 😄  - both are confusing to me

Reply:
It makes sense to me that, you see the current theme is light or dark (the page is that majority that color), and what you click is what you are choosing to swap to. 🤷‍♂️